src/server/game/AuctionHouse/AuctionHouseMgr.h

```diff
@@ -110,6 +110,7 @@ class AuctionHouseObject
     }
 
     typedef std::map<uint32, AuctionEntry*> AuctionEntryMap;
+    typedef std::unordered_map<ObjectGuid, time_t> PlayerGetAllThrottleMap;
```

Comment:
I understand the ease of this, but this storage is never cleared of records that are unused/useless/expired.